docs/topic-guides/elastic-infra/aws-spot.txt

```diff
@@ -0,0 +1,74 @@
+.. _aws-spot:
+
+####################
+ AWS Spot Instances
+####################
+
+This document describes how AWS spot instances work. Determined handles
```

Comment:
"This document describes how AWS spot instances work." -> "This document describes how to use AWS spot instances with Determined."

Can we also add a bit of context at the top of the doc explaining _why_ you might want to use spots with Determined?